修复一下 CI，虽然看上去不是你的 PR 造成的，不过 ci 还是得过

cocos/particle/animator/curve-range.ts:10:27 - error TS2304: Cannot find name 'CC_EDITOR'.

10 const SerializableTable = CC_EDITOR && [
                             ~~~~~~~~~

cocos/particle/animator/gradient-range.ts:23:27 - error TS2304: Cannot find name 'CC_EDITOR'.

23 const SerializableTable = CC_EDITOR && [

--- a/cocos/particle/animator/curve-range.ts
+++ b/cocos/particle/animator/curve-range.ts
@@ -6,8 +6,9 @@ import { ccclass, property } from '../../core/data/class-decorator';
 import { lerp } from '../../core/math';
 import { Enum } from '../../core/value-types';
 import { AnimationCurve } from '../../core/geometry';
+import { EDITOR } from 'internal:constants';
 
-const SerializableTable = CC_EDITOR && [
+const SerializableTable = EDITOR && [
     [ "mode", "constant", "multiplier" ],
     [ "mode", "curve", "multiplier" ],
     [ "mode", "curveMin", "curveMax", "multiplier" ],

--- a/cocos/particle/animator/gradient-range.ts
+++ b/cocos/particle/animator/gradient-range.ts
@@ -20,7 +20,7 @@ const GRADIENT_RANGE_MODE_RANDOM_COLOR = 2;
 const GRADIENT_RANGE_MODE_GRADIENT = 3;
 const GRADIENT_RANGE_MODE_TWO_GRADIENT = 4;
 
-const SerializableTable = CC_EDITOR && [
+const SerializableTable = EDITOR && [
     [ "_mode", "color" ],
     [ "_mode", "gradient" ],
     [ "_mode", "colorMin", "colorMax" ],

